notebooks/train_and_infer.ipynb

```diff
@@ -2,19 +2,161 @@
  "cells": [
```

Comment:
<p>this is to save the image, right? should be noticed, because it might look like the input path (which is created further down)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/grace/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>